samples/ComponentExamples/src/utils/loadConfigFromUrlQuery.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IdentityType } from './utils';
+
+export const loadConfigFromUrlQuery = (): IdentityType => {
+  const urlSearchParams = new URLSearchParams(window.location.search);
+  const params = Object.fromEntries(urlSearchParams.entries());
+
```

Comment:
See @JamesBurnside method to validate args as you parse them. It is a common error to load the test page with incorrect query args, and having a clear failure message really helps in debugging.

Reply:
Looks like it's here:
![image](https://user-images.githubusercontent.com/2684369/141211002-5ac51b72-4508-4f20-aae4-20e570af390c.png)